modules/administration/pages/ssl-certs-hsts.adoc

```diff
@@ -3,21 +3,33 @@
 
 HTTP Strict Transport Security (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security[HSTS]) is a policy mechanism that helps to protect websites against man-in-the-middle attacks such as protocol downgrade attacks and cookie hijacking.
 
+
 {productname} allows enabling HSTS, to enable it for a {productname} Server:
 
 .Procedure
-1. Edit `/etc/apache2/conf.d/zz-spacewalk-www.conf`
-2. Uncomment the line `# Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"`
+1. Create new configuration file in  `/etc/apache2/conf.d/<filename>.conf`, for example `/etc/apache2/conf.d/zzz-spacewalk-www.conf` or `/etc/apache2/conf.d/zz-spacewalk-www-custom.conf`.
```

Comment:
`/etc/apache2/conf.d/zzz-spacewalk-www.conf` is not a new file. We ship this file and it must not be changed or overwritten.

Reply:
@mcalmer - we ship zz-..., not zzz-... (triple z). I have added these two different options as an illustration, as well ass additional warning about file name creation.

Reply:
Uhhh, ok. Not sure how many customer might do the same error as me. They know that we have some files which start with "zz" but do not know how many "z". We can risk it, but we can also use "zzz-susemanager...." or "zzz-uyuni...." .
I think "susemanager" and "uyuni" are alphabetically **after** "spacewalk".
When it comes to sorting, this might be more specific

Reply:
Yes, it would be good to avoid "zzz" here.

Reply:
Thank you @mcalmer and @keichwa. Point taken, and I agree 100%. I had made the changes.